# Deactivate3Dscan.srv

# Request
---

# Response
bool done
